chain/client/src/client.rs

```diff
@@ -681,13 +680,13 @@ impl ClientActor {
             && self.last_block_processed.elapsed() < self.config.max_block_production_delay
```

Comment:
why not just change this `if false` here?
also add TODO with bug tracking explaining why we disabled it.